doc/crypto/RSA_get0_key.pod

```diff
@@ -46,7 +46,10 @@ RSA_set0_key() and passing the new values for B<n>, B<e> and B<d> as
 parameters to the function. Calling this function transfers the memory
 management of the values to the RSA object, and therefore the values
 that have been passed in should not be freed by the caller after this
-function has been called.
+function has been called.  The first time this function is called for
+a specific RSA object, B<n> and B<e> must be non-C<NULL>; however,
+this function may be called again with B<n> or B<e> being NULL to
+complete or set of values, typically to add a value for B<d>.
```

Comment:
Well the way you worded it in the some of the other pages was ok, so how about:
"The values B<n> and B<e> must be non-NULL the first time this function is called on a given RSA object. The value B<d> may be NULL. On subsequent calls any of these values may be NULL which means the corresponding RSA field is left untouched"